styles.css

```diff
@@ -0,0 +1,36 @@
+body {
+  font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+}
+
+.nav-bar {
+  display: flex;
+  justify-content: space-around;
+  padding-left: 30px;
+  flex: 1;
```

Comment:
- [ ] the colors are different to [see original](https://user-images.githubusercontent.com/61562626/129914258-eee7c795-21bb-4441-91fb-842bc8b281b2.jpeg)

- [ ] -  And your is 
![gfdggqd](https://user-images.githubusercontent.com/61562626/129914414-6035c696-7682-4803-b666-0c78b25c536a.jpeg)